pkg/client_data/lib/account_api.dart

```diff
@@ -49,6 +49,32 @@ class ClientSessionStatus {
   bool get isValid => expires != null && DateTime.now().isBefore(expires);
 }
 
+/// Account-specific information about a likes.
+@JsonSerializable()
+class PackageLikes {
```

Comment:
Maybe rename `LikedPackagesResponse` and update doc-comment to say something along the lines of:
> Response from listing of likes.